requirements.txt

```diff
@@ -1 +1,3 @@
+aubio
```

Comment:
All dependencies should be in requirements.txt so you can download and `pip install` the project requirements quickly.